src/Avalonia.Controls/DragDrop/DataObject.cs

```diff
@@ -0,0 +1,43 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace Avalonia.Controls.DragDrop
+{
+    public class DataObject : IDataObject
```

Comment:
This class doesn't seem to be used anywhere. Am I missing something?

Reply:
This class is an implementation which can be used to start dragging.
for example:

```csharp
DataObject dobj = new DataObject();
dobj.Set(DataFormats.Text, "Foobar");
var dragResult = await DragDrop.DoDragDrop(dobj, DragDropEffects.Copy|DragDropEffects.Move|DragDropEffects.Link);
```